configure

```diff
@@ -675,6 +680,11 @@ def configure_node(o):
   else:
     o['variables']['node_tag'] = ''
 
+  if options.release_urlbase:
+    o['variables']['node_release_urlbase'] = options.release_urlbase
```

Comment:
We can write this simply as `o['variables']['node_release_urlbase'] = options.release_urlbase or ''`. Should we even need the default value? Other options are not doing that.